backend/molgenis-emx2-rdf/src/test/java/org/molgenis/emx2/rdf/shacl/ShaclSelectorTest.java

```diff
@@ -63,7 +63,7 @@ void testFilteredShaclSet() {
   @Test
   void testAllNamesPresentInFiltered() {
     // does not test full file but only names for simplicity (1 shacl set is tested fully)
-    Set<String> expected = Set.of("fdp-v1.2", "dcat-ap-v3", "hri-v1", "hri-v2", "ejp-rd-vp");
+    Set<String> expected = Set.of("fdp-v1.2", "dcat-ap-v3", "hri-v1", "hri-v2", "hri-v2.0.2", "ejp-rd-vp");
```

Comment:
Ensure linting is done. Branch locally when building seems to have added a newline:
```java
Set<String> expected =
        Set.of("fdp-v1.2", "dcat-ap-v3", "hri-v1", "hri-v2", "hri-v2.0.2", "ejp-rd-vp");
```

If you do not want to build fully or add commit hook, simply run `./gradlew spotlessApply format` from project directory. You can probably undo the ones not related to your PR.

Reply:
okay, I ran gradlew spotlessApply format and pushed the resulting change in the file.